Source/Core/ApproximateTerrainHeights.js

```diff
@@ -38,15 +38,16 @@ var ApproximateTerrainHeights = {};
  * Initializes the minimum and maximum terrain heights
  * @return {Promise<void>}
  */
-ApproximateTerrainHeights.initialize = function () {
+ApproximateTerrainHeights.initialize = function (url) {
```

Comment:
@lilleyse Any thoughts on whether this is a good way to expose custom approximate terrain heights?

Reply:
What about something like `ApproximateTerrainHeights.defaultUrl = "myheights.json"`?

This mirrors `Ion.defaultAccessToken` which needs to be set before the viewer is created.

Not sure if this would get picked up by `createGroundPolylineWorker` though.